src/helpers/models.rs

```diff
@@ -126,7 +127,7 @@ pub struct SourceEvent {
     pub event: Event,
 
     /// A key to group sets of the events.
-    pub breakdown_key: PlainText,
+    pub breakdown_key: Number,
```

Comment:
Why is this stuff duplicated?

Reply:
I implemented event structures in models.rs for production IPA, i.e. secret shared and encrypted.

Synthetic event generation are in clear, hence we have another definition in gen_events.rs.